war/src/main/java/executable/Main.java

```diff
@@ -78,7 +78,7 @@ public class Main {
      * Sets custom session cookie name.
      * It may be used to prevent randomization of JSESSIONID cookies and issues like
      * <a href="https://issues.jenkins-ci.org/browse/JENKINS-25046">JENKINS-25046</a>.
-     * @since TODO
+     * @since 2.358
```

Comment:
```suggestion
     * @since 2.66
```

https://github.com/jenkinsci/extras-executable-war/commit/7afa0adbd7a6b0f1391e7ebd3213e4a79ae45e3f / https://github.com/jenkinsci/jenkins/commit/cb3f713122030b0ec5097b286d4e9e566a34a90a